### PR TITLE
Support for bmesh boolean and remove of that stuff that put error in console not sure why it was there lol

### PR DESCRIPTION
**Pull request by niewinny**

### CODE REVIEW

File: operators/cutters/complex_split_boolean.py [hunk 1]
```diff
@@ -77,18 +82,18 @@ def separate_mesh_by_looseparts(object):
...
     bpy.ops.object.mode_set(mode = 'OBJECT')
 
-def complex_split_boolean(target, cutters):
+def complex_split_boolean(target, cutters, use_bmesh):
     for cutter in cutters:
-        split_object(target, cutter)
+        split_object(target, cutter, use_bmesh)
 
-def split_object(object, cutter, margin = 0.0001):
+def split_object(object, cutter, use_bmesh, margin = 0.0001,):
```
Comment (JacquesLucke) [inline operators/cutters/complex_split_boolean.py]: I think here is a comma too much :D

File: ops_interactive.py [hunk 2]
```diff
@@ -146,8 +146,6 @@ def draw_callback_pb(self, context):
         blf.size(font_id, 12, get_dpi())
         blf.draw(font_id, "Standard is - " + str(is_bevel_2) + "      " + "Sstep is - " + str(is_bevel_3)+ "      "  + "CSharp is - " + str(is_bevel))
 
-    glDisable(GL_BLEND)
```
Comment (JacquesLucke) [inline ops_interactive.py]: why did you remove this?

Comment (niewinny) [inline ops_interactive.py]: it's different code right i does nothing

File: operators/cutters/complex_split_boolean.py (hunk 1 above)
Comment (niewinny) [inline operators/cutters/complex_split_boolean.py]: I don't know why.. will fix that

File: ops_interactive.py (hunk 2 above)
Comment (niewinny) [inline ops_interactive.py]: or am I totally wrong and we need it, I not sure now maybe that was important

Comment (JacquesLucke) [inline ops_interactive.py]: ok. it's not super important.
Just when you call glEnable for something you should call glDisable in the end. That's all. Not sure if that was the case here
